README.md

```diff
@@ -135,6 +134,17 @@ The controllers support several annotations on ingress resources. See the [examp
 feed has support for ALBs. Unfortunately, ALBs have a bug that prevents non-disruptive deployments of feed (specifically,
 they don't respect the deregistration delay). As a result, we don't recommend using ALBs at this time.
 
+# Comparison to official nginx ingress controller
+
+feed was started before the [official nginx ingress controller](https://github.com/kubernetes/ingress-nginx) became production ready. The main differences that exist now are:
+* feed has less features, as we only built it for our needs.
```

Comment:
Should be "fewer", rather than "less".

(Yes, I really am that pedantic sometimes.)